Resources/Textures/Clothing/Head/Soft/paramedicsoft.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e",
```

Comment:
it would be better if you linked the dmi file at this commit instead of the commit in general

Reply:
what do you mean?

Reply:
fairly sure every tg copyright is like this which is unfortunate, i agree with you

Reply:
Oh I think I know what you mean.

Reply:
I used the meta as a template for the file but the art is not from tgstation.  Should I just cut that totally and write a new copyright for SS14 like the one on the Dame Dane meta?  "Made by MagnusCrowe (Github) for SS14"